app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <string name="app_name">TaskRuler</string>
-    <string name="activtyName">Activity Name</string>
+    <string name="activityName">Activity Name</string>
```

Comment:
Another spelling mistake to ensure it still worked after changing the previous one.